common/shared/src/test/java/app/coronawarn/server/common/shared/util/HashUtilsTest.java

```diff
@@ -0,0 +1,32 @@
+package app.coronawarn.server.common.shared.util;
+
+import static app.coronawarn.server.common.shared.util.HashUtils.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import app.coronawarn.server.common.shared.util.HashUtils.MessageDigestAlgorithms;
+import com.google.protobuf.ByteString;
+import org.junit.jupiter.api.Test;
+import java.security.NoSuchAlgorithmException;
+
+class HashUtilsTest {
+
+  public static final String TEST_STRING = "string";
+
+  @Test
+  void testMd5DigestAsHex() {
+    String hash = md5DigestAsHex(TEST_STRING);
```

Comment:
Maybe we can make this test more useful. We could take an existing hex md5 hash (and its origin) and see if this function gives the same result.

Reply:
Done.